SampleApp/Settings.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Settings.swift
+//  ARChartsSampleApp
+//
+//  Created by Boris Emorine on 7/26/17.
+//  Copyright © 2017 Boris Emorine. All rights reserved.
+//
+
+import Foundation
+import ARCharts
+
+struct Settings {
+    
+    var animationType: ARChartPresenter.AnimationType = .fade
+    var longPressAnimationType : ARChartHighlighter.AnimationStyle = .shrink
+    var barOpacity: Float = 1.0
+    var labels = true
+    var numberOfSeries = 10
+    var numberOfIndices = 10
+    var graphWidth: Float = 0.3
+    var graphHeight: Float = 0.3
+    var graphLength: Float = 0.3
+    var dataSet: Int = 0
+    
+    public func index(forEntranceAnimationType animationType: ARChartPresenter.AnimationType?) -> Int {
+        guard let animationType = animationType else {
+            return 0
+        }
+        
+        switch animationType {
+        case .fade:
+            return 0
+        case .progressiveFade:
+            return 1
+        case .grow:
+            return 2
+        case .progressiveGrow:
+            return 3
+        }
```

Comment:
Nit: These could be `rawValue`s on the `ARChartPresenter.AnimationType` enum, then this function wouldn't be necessary.